### PR TITLE
fix(environments): Fix project dashboard environment filtering

### PR DESCRIPTION
**Pull request by lynnagara**
Ensure environment filter is applied

### CODE REVIEW

**Review by MaxBittker — APPROVED** [review]
looks good 👍

File: src/sentry/static/sentry/app/views/projectDashboard.jsx [hunk 1]
```diff
@@ -36,12 +36,18 @@ const ProjectDashboard = createReactClass({
...
+      ),
       ...this.getQueryStringState(),
     };
   },
 
   componentWillMount() {
     this.props.setProjectNavSection('dashboard');
+
+    // Manually fire onLatestContextChange
+    this.onLatestContextChange(LatestContextStore.getInitialState());
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/projectDashboard.jsx]: shouldn't this go in getInitialState though?

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/projectDashboard.jsx]: It could, but can't fire `setState` in there, so would prefer to keep it here

Comment (billyvg) [reply src/sentry/static/sentry/app/views/projectDashboard.jsx]: Wouldn't `getInitialState().activeEnvironment` == `(hasEnvironmentsFeature) && LatestContextStore.getInitialState().environment`

**Pull request merged**